R/quosure-get_code.R

```diff
@@ -20,3 +21,13 @@ setGeneric("get_code", function(object) {
 setMethod("get_code", signature = "Quosure", function(object) {
   object@code
 })
+
+#' @rdname get_code
+#' @export
+setMethod("get_code", signature = "quosure.error", function(object) {
```

Comment:
I wonder if there will be any downstream consequences of this (i.e. if you get_code and then do something with it maybe this should also return a quosure.error but with the extra code in the conditionMessage ?

Reply:
changed this to return a quosure.error with the trace included